extensions/wikia/Captcha/Factory/Module.class.php

```diff
@@ -48,6 +48,10 @@ public static function verifyCaptchaClass( $captchaClass ) {
 	 * Load Captcha.js on demand when we're getting an instance of a Captcha.
 	 */
 	public static function addCaptchaJS() {
-		\Wikia::addAssetsToOutput( 'captcha_js' );
+		// Make sure we only load FancyCaptcha library once
+		if ( !\F::app()->wg->CaptchaLibraryLoaded ) {
```

Comment:
Looks like a php [static variable](http://php.net/manual/en/language.variables.scope.php#language.variables.scope.static) could be used here instead of a global